package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@catena-x/portal-shared-components",
-  "version": "2.0.10",
+  "version": "2.0.11",
```

Comment:
I'm uncertain about raising the path level of the components due to a action upgrade in a GH workflow.
It maybe worth looking into limiting the triggering of the release workflow to certain paths, I paste an example from the backend: https://github.com/eclipse-tractusx/portal-backend/blob/dev/.github/workflows/administration-service.yml#L24

Reply:
done